pkgs/applications/misc/buku/default.nix

```diff
@@ -1,35 +1,38 @@
 { stdenv, python3, fetchFromGitHub, fetchpatch }:
 
 with python3.pkgs; buildPythonApplication rec {
-  version = "3.7";
+  version = "3.8";
   pname = "buku";
 
   src = fetchFromGitHub {
     owner = "jarun";
     repo = "buku";
     rev = "v${version}";
-    sha256 = "0qc6xkrhf2phaj9fhym19blr4rr2vllvnyljjz909xr4vsynvb41";
-  };
-
-  patches = fetchpatch {
-    url = https://github.com/jarun/Buku/commit/495d6eac4d9371e8ce6d3f601e2bb9e5e74962b4.patch;
-    sha256 = "0py4l5qcgdzqr0iqmcc8ddld1bspk8iwypz4dcr88y70j86588gk";
+    sha256 = "0gv26c4rr1akcaiff1nrwil03sv7d58mfxr86pgsw6nwld67ns0r";
   };
 
   checkInputs = [
     pytestcov
-    pytest-catchlog
     hypothesis
     pytest
     pylint
     flake8
+    pyyaml
   ];
 
   propagatedBuildInputs = [
     cryptography
     beautifulsoup4
     requests
     urllib3
+    flask
+    flask_api
+    flask_bootstrap
+    flask_paginate
+    flask_wtf
```

Comment:
@Mic92 Also look at these deps here, would be weird for some of them to have a "-"

Reply:
There are many counter example also in python packages. I am not sure how consistent this was done for flask.